test/util_test.py

```diff
@@ -25,3 +29,18 @@ def test_accepted_unit(self):
 
         with pytest.raises(TypeError):
             parse_unit("1:ul", ["second", "kg"])
+
+class TestUtil(object):
+    def test_compatible_reservoir_container(self):
```

Comment:
I would like to see one more test that we _can_ stamp between 384-well plates and single-well reservoirs